Update documentation to clarify ES node.processors section. (#5941)
elastic/cloud-on-k8s: docs/orchestrating-elastic-stack-applications/managing-compute-resources.asciidoc

[id="{p}-elasticsearch-cpu"]
==== CPU resources

> Although `node.processors` isn't directly set by the ECK Operator, the value set for CPU limits or requests directly impacts the Elasticsearch `node.processors` setting as, by default the number of processors is automatically detected, and the thread pool settings are automatically set. The following table gives the default value for `node.processors` given the cpu limits and requests set on the `elasticsearch` container:

Reviewer: Or make it shorter:
```suggestion
The value set for CPU limits or requests directly impacts the Elasticsearch `node.processors` setting. By default Elasticsearch automatically detects the number of processors and sets the thread pool settings based on it. The following table gives the default value for `node.processors` given the CPU limits and requests set on the `elasticsearch` container:
```
